Replace startswith and endswith with stdlib calls

contificate opened the pull request:
These functions were introduced in OCaml 4.13.

psafont commented:
Coveralls upload is failing for some strange reason

freddy77 commented:
The code is good. The commit message a bit misleading. Maybe `Replace startswith and endswith with standard Ocaml functions` ?

contificate commented:
> The code is good. The commit message a bit misleading. Maybe `Replace startswith and endswith with standard Ocaml functions` ?

Fair point; I've rewritten the commit message and renamed this PR (so any merge commit isn't inconsistently named).
